pytest_splinter/plugin.py

```diff
@@ -225,7 +236,7 @@ def splinter_make_screenshot_on_failure(request):
 @pytest.fixture(scope='session')  # pragma: no cover
 def splinter_screenshot_dir(request):
     """Browser screenshot directory."""
-    return os.path.abspath(request.config.option.splinter_screenshot_dir)
+    return request.config.hook.pytest_splinter_screenshot_dir(config=request.config)[0]
```

Comment:
why does it take the first?, this is strange

Reply:
I misunderstood how hooks store values. I've added the hookspec decorator with firstresult=True to address this.